### PR TITLE
summary section

### PR DESCRIPTION
**Pull request by Yshika**

### CODE REVIEW

File: src/components/Nav/style.scss [hunk 1]
```diff
@@ -3,10 +3,18 @@
...
+  width: 100%;
+  top: 0;
 
   &-title {
     @include heading;
+    font-weight: 400;
+
+    &:hover {
+      text-decoration: none;
+    }
```
Comment (saksham-johri) [inline src/components/Nav/style.scss:20]: need of this?

File: src/components/Nav/style.scss [hunk 2]
```diff
@@ -26,7 +34,7 @@
     display: none;
   }
 
-  @include responsive('small') {
+  @include responsive('medium') {
     padding: 2rem 2rem;
```
Comment (saksham-johri) [inline src/components/Nav/style.scss]: can write `padding: 2rem`

File: src/components/Summary/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        <img src='/profile_pic.jpg' alt='profile' />
+      </div>
+      <div className='info'>
+        <h3 className='greeting'>Hello I'm</h3>
+        <h1 className='name'>{data?.name}</h1>
+        <h1 className='job'>{data?.job}</h1>
+
+        <div className='cta-container'>
+          <button>Download CV</button>
+          <button className='cta-dark'>Contact Info</button>
```
Comment (saksham-johri) [inline src/components/Summary/index.tsx]: variant of btn?

File: src/components/Summary/style.scss [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      }
+    }
+
+    .social-links {
+      display: flex;
+      gap: 1rem;
+
+      .icon {
+        width: 2rem;
+        height: 2rem;
```
Comment (saksham-johri) [inline src/components/Summary/style.scss:47]: if you are giving width & height here, thn why give `width: 100%` & `height: 100%` in global CSS?

Comment (Yshika) [reply src/components/Summary/style.scss:47]: its div of img

File: src/components/About/index.tsx [hunk 5]
```diff
@@ -1,10 +1,43 @@
...
-      <h1>About Me</h1>
-      <p>This is the about section of the portfolio.</p>
+      <div className='heading-container'>
+        <h3 className='know-more'>Get To Know More</h3>
+        <h1 className='heading'>About Me</h1>
+      </div>
+
+      <div className='about-content'>
+        <div className='about-pic'>
+          <img src='/profile_pic.jpg' alt='profile' />
```
Comment (saksham-johri) [inline src/components/About/index.tsx]: image src should come from `data.json`

File: src/components/About/style.scss [hunk 6]
```diff
@@ -0,0 +1,96 @@
+@import '../../styles/index.scss';
+
+.about {
+  min-height: 100dvh;
```
Comment (saksham-johri) [inline src/components/About/style.scss]: use can use `section` tag, and can give default CSS properties to `section`

File: src/components/About/style.scss [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+    .about-pic {
+      width: 22rem;
+      height: 22rem;
+      border-radius: 20px;
+      overflow: hidden;
+
+      img {
+        filter: grayscale(100%);
+      }
```
Comment (saksham-johri) [inline src/components/About/style.scss]: shouldn't use tags to provide css

File: src/components/About/style.scss [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+    .about-overview {
+      width: 50%;
+      display: grid;
+      gap: 2rem;
+      justify-content: center;
+
+      p {
+        @include paragraph;
+      }
```
Comment (saksham-johri) [inline src/components/About/style.scss]: shouldn't use tags to provide css

File: src/components/Summary/style.scss [hunk 9]
```diff
@@ -0,0 +1,62 @@
+@import '../../styles/index.scss';
+
+.summary {
+  display: flex;
+  align-items: center;
+  height: 100dvh;
```
Comment (saksham-johri) [inline src/components/Summary/style.scss]: use can use `section` tag, and can give default CSS properties to `section`

**Review by saksham-johri — CHANGES_REQUESTED** [review]

File: src/styles/_typography.scss [hunk 10]
```diff
@@ -1,12 +1,31 @@
...
+  color: $paragraph_color;
+}
+
+@mixin box {
+  border: 1px solid black;
+  border-radius: 1.25rem;
+  padding: 2rem;
+  justify-items: center;
+  width: 50%;
+}
```
Comment (saksham-johri) [inline src/styles/_typography.scss]: should be in mixins

**Review by saksham-johri — APPROVED** [review]

**Pull request merged**